lib/Xmds/Soap.php

```diff
@@ -1476,48 +1476,32 @@ protected function doSubmitStats($serverKey, $hardwareKey, $statXml)
             $duration = $node->getAttribute('duration');
             $count = $node->getAttribute('count');
 
-            // if fromdt and to dt are same then ignore them
-            if ($fromdt == $todt) {
-                $this->getLog()->debug('Fromdt (' . $fromdt. ') and ToDt (' . $todt. ') are same. ');
+            if ($fromdt == '' || $todt == '' || $type == '') {
+                $this->getLog()->error('Stat submitted without the fromdt, todt or type attributes.');
```

Comment:
Because there is nothing we can do about this, I think we should downgrade these log levels to `info` - it saves filling the log with errors we cannot do anything about.